com.freerdp.FreeRDP.json

```diff
@@ -0,0 +1,177 @@
+{
+    "app-id": "com.freerdp.FreeRDP",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "build-options" : {
+        "cflags": "-O3",
+        "cxxflags": "-O3"
```

Comment:
why do you need to override the -O flags?

Reply:
since we process a lot of bitmap data having better optimization flags set really improves performance. (specifically the simdi intrinsics that are automatically used with higher optimization flags)